### ISSUE TITLE
passing height arguments to \GreNABCNeumes

### ISSUE THREAD

**Issue by eroux** [gregorio-project/gregorio#1181]
I think that in order to allow more interesting vertical placement of nabc neumes, it would be necessary to have two more arguments to `\GreNABCNeumes` which would be the highest and lowest height of the notes (or signs) of the square notes it corresponds to. This would allow two things:

- if, on a line, only one note is at a very high pitch, not all nabc neumes need to be raised, only those corresponding to this note
- in some layouts, the nabc notes are always just above the notes, even if it's inside the staff lines

Once this is done we can discuss more in details about the kind of layout and TeX interfaces we want, but this is a first step we'll have to make anyway...

**Comment by henryso** [comment]
Closing this.  Let's open a new issue with the details on TeX adjustments once that's worked out.
